torch/csrc/lazy/core/lazy_graph_executor.cpp

```diff
@@ -475,6 +482,8 @@ void LazyGraphExecutor::MarkStep(const BackendDevice& device) {
   DeviceContextArena::Get()->MarkStep(device);
   ScopePusher::ResetScopes();
   g_tls_data.Reset();
+  Trie::Get()->ResetCurrent();
```

Comment:
I guess it would be nicer to name the class such that when you call this function it's more apparent what the thing is used for.  LazyTraceCache?